.github/workflows/drizzle_ci.yml

```diff
@@ -0,0 +1,81 @@
+name: Drizzle Package Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Python 3.9 Test
+            pyver: 3.9
+            dependencies: 
+
+          - name: Old Dependencies
+            pyver: 3.6.10
+            dependencies: |
+              python -m pip install numpy~=1.16.0
+
+          - name: Development Build
+            pyver: 3.8.5
+            dependencies: |
+              python -m pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple --pre -I numpy
+              python -m pip install git+https://github.com/astropy/astropy
+    steps:
+    - uses: actions/checkout@v2
+    - name: ${{ matrix.name }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.pyver }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .[test]
+        ${{ matrix.dependencies }}
+    - name: Test with Pytest
+      run: |
+        pytest
+
+  doc_build:
+    name: Document Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        sudo apt-get install graphviz texlive-latex-extra dvipng
+        python -m pip install --upgrade pip
+        python -m pip install -e .[docs]
+    - name: Build documents with sphinx
+      run: |
+        cd docs
+        sphinx-build . ./_build
+
+  lint:
+    name: Lint with flake8
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8
+    - name: Build documents with sphinx
```

Comment:
Update this name description to be linting-related.  "Run flake8".  Also, I don't think this is doing anything, as it is looking for `asdf`, not anything in `drizzle`.

Reply:
I changed the name and changed the test to run on `drizzle`, instead of `asdf`.